cmd/containerd/command/main_windows.go

```diff
@@ -54,12 +54,12 @@ func handleSignals(ctx context.Context, signals chan os.Signal, serverC chan *se
 					log.G(ctx).WithError(err).Error("notify stopping failed")
 				}
 
-				if server == nil {
-					close(done)
-					return
+				if server != nil {
+					server.Stop()
 				}
-				server.Stop()
+				cancel()
```

Comment:
Is it possible to align with main_unix.go? The main_unix.go cancels first and then stop the server.

Reply:
See the previous conversation with @crosbymichael

Reply:
Yeah. But I feel confused about that comment. 
The last commit only changes the `cancel` in `main_unix.go`, not including `main_windows.go`. 
So I want to confirm that :)

Reply:
Ah, I see. Fixed this.